doc/05-Howto-Jetstream.md

```diff
@@ -227,5 +227,5 @@ Submit your manuscript for publication, assured that data provenance and workflo
 
 
 ## Author information
-Volker Brendel (vbrendel@indiana.edu)
+Volker Brendel (vbrendel@indiana.edu)  
```

Comment:
The two spaces at the end of the line insert a line break, which is what I believe was intended.